fractal_server/app/security/__init__.py

```diff
@@ -196,6 +194,7 @@ def get_jwt_strategy() -> JWTStrategy:
     return JWTStrategy(
         secret=settings.JWT_SECRET_KEY,  # type: ignore
         lifetime_seconds=settings.JWT_EXPIRE_SECONDS,
+        token_audience=["fractal"],
```

Comment:
Maybe remove this?
Ref https://mojoauth.com/glossary/jwt-audience

Reply:
By removing this we get the default
```
"aud": [
    "fastapi-users:auth"
]
```